src/ReactDataGrid.js

```diff
@@ -857,7 +857,7 @@ const ReactDataGrid = React.createClass({
   },
 
   dragEnabled: function(): boolean {
-    return this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined;
+    return (this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined) && this.props.enableDragAndDrop !== false;
```

Comment:
Hey @thomas-shares.

Enabling by default won't do any warm as we also check for the callbacks.

You can set `enableDragAndDrop` to true in the `defaultProps`